src/main/java/uk/co/ramp/covid/simulation/lockdown/LocalLockdownEventGenerator.java

```diff
@@ -0,0 +1,92 @@
+package uk.co.ramp.covid.simulation.lockdown;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import uk.co.ramp.covid.simulation.population.Person;
+import uk.co.ramp.covid.simulation.util.Time;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LocalLockdownEventGenerator extends LockdownEventGenerator {
+    private static final Logger LOGGER = LogManager.getLogger(LocalLockdownEventGenerator.class);
+
+    private Double newCasesThreshold = null;
+    private Integer numHospitalisedThreshold = null;
+
+    private Double socialDistance = null;
+    
+    private int prevInfected = 0;
+    private boolean gotInitialInfections = false;
+
+    private int getNumInfected() {
```

Comment:
Could we use DailyStats, rather than calculating these sums here?

Reply:
Possibly - but it's a little tricky (do-able, but messy) to get it into the generator (especially since it's really *yesterdays* stats we want). For now I suggest we leave it as is, just for simplicity, but if we write a new generator that also needs similar stats we can work out a way to add this in.